glide.yaml

```diff
@@ -55,3 +55,7 @@ import:
   subpackages:
   - unix
 - package: github.com/samalba/dockerclient
+- package: github.com/samuel/go-zookeeper
+  version: 8c6a1d79a44d2742da3660540d1fd43f4ddcfc8c
+  subpackages:
+    - zk
```

Comment:
OK, got it at last. I’ll update the PR once I get `make test` pass (see https://github.com/joyent/containerpilot/pull/214/files#r79051645) so that Travis can be happy as well.